README.md

```diff
@@ -228,5 +228,9 @@ docker run --rm -it \
 Then navigate to http://localhost:8080 .
 
 ### Changelog
+ - v1.0.0 - **Braking Changes**
+    - Search traces by tags support
+    - Use static Jaeger components version (1.18)
+        - Since tchannel has been deprecated in version 1.16, when running standalone Jaeger agent, use grpc reporter protocol - see "Run Jaeger agent"
```

Comment:
```suggestion
 - v1.0.0 - **Braking Changes**
    - Support for searching traces by tags is affected by the introduction of new tags.
    - Use of static Jaeger components version (1.18)
        - Since the deprecation of TChannel in version 1.16, it is necessary to use gRPC reporter protocol when running a standalone Jaeger agent. - see "Run Jaeger agent"
```